src/gsModeling/gsPlanarDomain.h

```diff
@@ -21,6 +21,7 @@
 
 #include <gsModeling/gsCurveLoop.h> 
 #include <gsModeling/gsTemplate.h>
+#include <gsUtils/gsMesh/gsMesh.h>
```

Comment:
Does it work if you include gsMesh.h in gsPlanarDomain.hpp ? This would be more preferable

Reply:
It's then needed in gsTrimSurface.h

Yesterday I build it without include and with memory::unique_ptr<gsMesh<T> > instead of any gsMesh<T>::uPtr in gsPlanar and gsTrimSurface. It took on my computer about 18s longer (r: 6m27.940s, u:17m16.519s vs. 6m35.502s, u:17m27.351s) to build.